main.py

```diff
@@ -0,0 +1,78 @@
+from transformers import RobertaForMaskedLM
+from transformers.models.roberta.modeling_roberta import RobertaPooler
+from torchvision.models import resnet18
+import torch.nn as nn
+import torch 
+from transformers.data.data_collator import DataCollatorForLanguageModeling
+import pytorch_lightning as pl 
+from utils.model_utils import freeze_everything_except_top
+import random 
+
+class JeopardyModel(pl.LightningModule):
+    def __init__(self, config, pretrained_bert):
+        super().__init__()
+        # TODO: freeze everything except the last 3 layers of Roberta
+        self.model = RobertaForMaskedLM.from_pretrained('roberta-base')
+        freeze_everything_except_top(self.model, 3)
+        self.word_embeddings = self.model.roberta.embeddings.word_embeddings
+        self.image_feature_extractor = resnet18(pretrained=False)
+        self.image_feature_extractor.fc = nn.Linear(512, self.word_embeddings.embedding_dim) 
+        self.alpha = config.model_params.alpha
```

Comment:
what is alpha?

Reply:
I thought we wanted the final loss to be a*contra_loss + (1-a)*mlm_loss

Reply:
i see,add comments!